buildenv/jenkins/Jenkinsfile_personal

```diff
@@ -1,14 +1,15 @@
 #!groovy
+def JAVA_IMPL='hotspot'
 pipeline {
     agent {label "${params.MACHINE_LABEL}"}
     parameters {
     	string (defaultValue: "AdoptOpenJDK", description: 'personal repo', name: 'personalRepo')
     	string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
 		choice (choices: 'x64_linux\nx64_mac\ns390x_linux\nppc64le_linux\naarch64_linux', description: 'PLATFORM?', name: 'PLATFORM')
 		choice (choices: 'SE90\nSE80\nSE100', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
-		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8\nopenjdk10\nopenjdk10-openj9', description: 'What is JVM Version?', name: 'JVM_VERSION')
+		choice (choices: 'openjdk9-openj9\nopenjdk8-openj9\nopenjdk9\nopenjdk8\nopenjdk10\nopenjdk10-openj9\nopenjdk10-sap', description: 'What is JVM Version?', name: 'JVM_VERSION')
```

Comment:
it will be nice to list the choices in order openjdk8, openjdk8-openj9, openjdk9, openjdk9-openj9, openjdk10, openjdk10-sap

Reply:
Updated.